### DISCUSSION TITLE
Dtu settings : Missing values

### DISCUSSION THREAD

**Discussion by redridinghood314** [tbnobody/OpenDTU#2297] [Q&A]
Hello ! 
First thanks for the work on opendtu. This IS really great project !
Here my problem , i have changed transmit power to try to extend range and disocver my new inverter. Then i have upgraded firmware (from 24.5.6 (if i remember well) ) to 24.9.21. 
My New inverter is now working well, so i would like to see if i Can decrease transmit power, but when i change the value and save it indicates that some value are Missing, i do not understand why. Could you please help me on that ?thanks a lot !

**Comment by tbnobody** [comment]
Found the issue. will be fixed in the next release.

**Reply by redridinghood314** [reply]
Thanks a lot for this Quick return !!
May i ask you  one question, my new inverter  (hoymiles HMS 1000) is showing unknown event id 23 many Time (but seems to have stopped), do you know what is it or where i can search for explanation ?thanks !!

**Comment by tbnobody** [comment]
> do you know what is it or where i can search for explanation ?

Unfortunatly not. All descriptions which are known are already shown. Only possibility would be the device manual (the previous id's where read from the manual)

**Reply by redridinghood314** [reply]
Ok, thanks, unfortunately error code starts at 121 in product documentation, i will try to contact hoymiles and give feedback to add it if i get an answer. Thanks !

**Reply by stefan123t** [reply]
@redridinghood314 the most concise list we found in warn_code.json of the S-Miles Installer Apps APK file. 

https://github.com/tbnobody/OpenDTU/issues/121#issuecomment-1752124208

Maybe they have added the alarm / event code the list lately? I have not checked or verified this since quite some time.

**Comment by tbnobody** [comment]
> I have not checked or verified this since quite some time.

I've just checked versioin 1.1.29... Nothing regarding ID 23

**Reply by stefan123t** [reply]
Thanks we could only send a fake Event log to the S-Miles Cloud using the protobuf API that @rovo89 @DennisOSRM and others built.
Such a fake Event log may contain multiple of the yet unknown Event IDs like 23 and we could then check in the S-Miles Cloud how these unknown Event IDs get translated / interpreted.

**Comment by Zephalon** [comment]
I have the same issue in 24.9.22. A workaround is to export the config.json, adjust the values and re-import the file.

**Reply by tbnobody** [reply]
Thats absolutly ok as this issue was fixed in 24.9.23 and you are using an older version.
